editor/src/components/canvas/remix/remix-utils.tsx

```diff
@@ -99,3 +128,405 @@ export function getRoutesFromRouteManifest(
     return []
   }
 }
+
+export function createAssetsManifest(routes: RouteManifest<EntryRoute>): AssetsManifest {
+  return {
+    entry: { imports: [], module: 'TODO' },
```

Comment:
What is the actual TODO here? I don't think this is something we're tracking anywhere